weareinreach/inreach-catalog
# WIP - localization updates for Badge, DetailsServiceType,DetailServices,etc

## Description
while testing the local switching found a few more places that needed the language keys added to the various code elements.
Badge, DetailsServiceType,DetailServices,ResourceListItems, ResourceTypeSelector, tags

updated language files to include several missing keys

## Asana ticket:
https://app.asana.com/0/1132189118126148/1202000181276366

## PR Checklist




## How to Test
The localization changes are just some of the changes needed In order to fully translate the catalog. We will also need to disable the ability to choose English or Spanish under the google translate options - see asana ticket - https://app.asana.com/0/1132189118126148/1188052226111153. This ticket is in the backlog until the translation/localization work is completed. (note, there could be more as yet defined changes as well). Therefore In order to see all of the translated text from this PR, some settings will need to be set manually. In order to see the AC native text from this PR, as opposed to google translated text, the ‘lang’ and ‘locale’ LOCAL storage values need to be set manually via developer tools.

to verify the correctly translated values form this PR are displayed correctly,

1. go to the AsylumConnect home screen,
2. click on ‘choose a different country’
3. select ‘English’ for the language
4. select ‘united states’ for the country
5. open developer tools
6. under LOCAL storage, set ‘lang’ to ‘Español’, set ‘locale’ to ‘es_MX’
7. refresh the page
8. go to the various sections of the app that have been updated (see below) and verify the AC specified text appears.

<img width="451" alt="Screen Shot 2022-02-14 at 4 36 41 PM" src="https://user-images.githubusercontent.com/10123216/153973080-6d2e5b4a-a777-491b-9aa5-929e280d0da3.png">

components to verify:
Badge, DetailsServiceType,DetailServices,ResourceListItems, ResourceTypeSelector, tags

check desktop view as well as mobile view
